src/generator/templates/Cargo.toml.mako

```diff
@@ -31,6 +31,7 @@ hyper-rustls = "0.25.0"
 ## Must match the one hyper uses, otherwise there are duplicate similarly named `Mime` structs
 mime = "^ 0.3.0"
 serde = { version = "^ 1.0", features = ["derive"] }
+utoipa = { version = "^4.2", features = ["axum_extras", "uuid", "chrono", "url"] }
```

Comment:
There is indeed no way this would be added without being optional, for use with a feature toggle. Putting this in should be no problem though.

Reply:
That's why it's a draft, I just did it hard and fast to see if it actually works for my use case.